contexts/server-base/requirements.txt

```diff
@@ -19,6 +19,7 @@ java-utilities
 
 # optional autocomplete feature
 jedi==0.19.1
+docstring_parser==0.16
```

Comment:
We typically prefer to be more lenient with our "requirements"; `jedi` is a special case where we have seen bad things happen with different versions, it's the only version we pin in `py/server/setup.py`. Do you have reason to believe that earlier versions of docstring_parer won't work? Or, that future versions of docstring_parser will be incompatible? If not, it would be preferable to not set a version (or, if you know old versions don't work, set a minimum version).

Reply:
Updated it to be `>=0.16`. I'm using a feature that's new in this version and there's no reason to believe that future versions would be incompatible.